runtime/Makefile

```diff
@@ -50,6 +50,7 @@ integ-test:
 			--volume /sys:/sys \
 			--volume /run/udev/control:/run/udev/control \
 			--volume $(CURDIR)/logs:/var/log/firecracker-containerd-test \
+			--volume $(CURDIR)/scripts:/tmp/scripts \
```

Comment:
I think it may be more consistent to `COPY` this into the Dockerfile we use for the test images, around here: https://github.com/firecracker-microvm/firecracker-containerd/blob/master/tools/docker/Dockerfile#L180

Reply:
Thanks. Will do.

Reply:
Addressed https://github.com/firecracker-microvm/firecracker-containerd/pull/233/commits/c9f3074909d63e05909092f6f69923b89d2b3ce5